app/controllers/dashboards_controller.rb

```diff
@@ -14,9 +15,12 @@ def show
                      .joins(collection: :reviewed_by)
                      .where('reviewers.user_id' => current_user),
       in_progress: Work.with_state(:first_draft, :version_draft, :rejected)
-                       .where(depositor: current_user)
+                       .where(depositor: current_user),
+      collection_managers_in_progress: Collection.with_state(:first_draft, :version_draft)
+                                                 .joins(:managers).where("managers.user_id = #{current_user.id}")
```

Comment:
is there a better way to scope the query for collections where the current user is marked as the collection manager? or does this look ok?

Reply:
this is fine, but you should use bind variables. See https://blog.saeloun.com/2019/10/28/bind-parameters-in-activerecord-sql-queries.html

Reply:
Alternatively `current_user.manages_collections.with_state(:first_draft, :version_draft)`

Reply:
thanks, i am familiar with but forgot about bind variables for this case, but that habtm association format looks nice